examples/02_decoding/plot_miyawaki_encoding.py

```diff
@@ -0,0 +1,259 @@
+# activity in this voxel.
+
+from sklearn.linear_model import Ridge
+from sklearn.cross_validation import KFold
+
+##############################################################################
+# Using 10-fold cross-validation, we partition the data into 10 'folds',
+# we hold out each fold of the data for testing, then fit a ridge regression
+# to the remaining 9/10 of the data, using y_train as predictors
+# and X_train as targets, and create predictions for the held-out 10th.
+from sklearn.metrics import r2_score
+
+estimator = Ridge(alpha=100.)
+cv = KFold(len(y_train), 10)
+
+scores = []
+for train, test in cv:
+    # we train the Ridge estimator on the training set
+    # and predict the fMRI activity for the test set
+    predictions = Ridge(alpha=100.).fit(
+    y_train.reshape(-1, 100)[train], X_train[train]).predict(
+        y_train.reshape(-1, 100)[test])
+    # we compute how much variance our encoding model explains in each voxel
+    scores.append(r2_score(X_train[test], predictions,
+                           multioutput='raw_values'))
+
+##############################################################################
+# Mapping the encoding scores on the brain
+# ----------------------------------------
+# To plot the scores onto the brain, we use :func:`nilearn.image.new_img_like`
+# to create a Nifti1Image containing the scores and then threshold it:
+
+from nilearn.image import threshold_img, new_img_like
+from nilearn.masking import unmask
+
+cut_score = np.array(scores).mean(axis=0)
+cut_score[cut_score < 0] = 0
+
+# bring the scores into the shape of the background brain
+score_map_img = new_img_like(dataset.background,
+                         unmask(cut_score, dataset.mask).get_data())
+
+thresholded_score_map_img = threshold_img(score_map_img, threshold=1e-6)
+
+##############################################################################
+# Plotting the statistical map on a background brain, we mark four voxels
+# which we will inspect more closely later on.
+from nilearn.plotting import plot_stat_map
+from nilearn.image.resampling import coord_transform
+
+def index_to_xy_coord(x, y, z=10):
+    '''Transforms data index to coordinates of the background + offset'''
+    coords = coord_transform(x, y, z,
+                             affine=thresholded_score_map_img.get_affine())
+    return np.array(coords)[np.newaxis, :] + np.array([0, 1, 0])
+
+
+xy_indices_of_special_voxels = [(30, 10), (32, 10), (31, 9), (31, 10)]
+
+display = plot_stat_map(thresholded_score_map_img, bg_img=dataset.background,
+                        cut_coords=[-8], display_mode='z', aspect=1.25,
+                        title='Explained variance per voxel')
+
+# creating a marker for each voxel and adding it to the statistical map
+
+for i, (x, y) in enumerate(xy_indices_of_special_voxels):
+    display.add_markers(index_to_xy_coord(x, y),
+                        edgecolor=['b', 'r', 'magenta', 'g'][i],
```

Comment:
I am stupid: you told me why no cyan: no corresponding colormap. It's before coffee, sorry.

How about black, then? You can use the gray colormap.